pages/common/ipfs.md

```diff
@@ -0,0 +1,32 @@
+# ipfs
+
+> Inter Planetary File System.
+> A peer-to-peer hypermedia protocol to make the web faster, safer, and more open.
+
+- Save a remote file and give It a name but not pin It:
+
+`ipfs get {{hash}} -o {{filename}}`
+
+- Add a file from local to the file system with the filename (-w) and pin It:
```

Comment:
Is there a need to mention "-w" ? If you think so, we also need to explain the flag, which is "wrap with directory".

We should also mention that this command prints the file hash which is necessary in the first command.

Reply:
To simplify is maybe better avoiding the -w flag.. actually It's not necessarily needed.. just a fashion :)